### PR TITLE
New company suggestion: Bigpoint GmbH

### PR DESCRIPTION
**Pull request by Richardds**
Hello,
I did not find the Bigpoint GmbH on your list.

Cheers,
Richard

### CODE REVIEW

File: companies/bigpoint.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        "pt",
+        "ro",
+        "sk",
+        "se"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Bigpoint GmbH",
+    "address": "Drehbahn 47-48\n20354 Hamburg\nDeutschland",
```
Comment (baltpeter) [inline companies/bigpoint.json:27]: As this record applies to various countries, the country should be in English.

File: companies/bigpoint.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    "phone": "+49 40 8814130",
```
Comment (baltpeter) [inline companies/bigpoint.json:28]: The privacy policy lists a phone specific to privacy-related requests (+49 821 90786458). Please use that instead ([see](https://github.com/datenanfragen/data/#data-sources)).

File: companies/bigpoint.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    "fax": "+49 40 881413 906",
```
Comment (baltpeter) [inline companies/bigpoint.json:29]: I've probably overlooked something but where did you find that fax number?

File: companies/bigpoint.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    "email": "privacy@bigpoint.net",
+    "web": "https://www.bigpoint.net/",
+    "sources": [
+        "https://legal.bigpoint.com/DE/privacy-policy/de-DE",
+        "https://legal.bigpoint.com/DE/terms-and-conditions/de-DE",
+        "https://legal.bigpoint.com/DE/imprint/de-DE"
```
Comment (baltpeter) [inline companies/bigpoint.json:35]: From what I've seen, the latter two sources don't have any new information that isn't already covered in the privacy policy. If so, you don't need to add them.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the record! I have left a few comments below.

File: companies/bigpoint.json (hunk 3 above)
Comment (Richardds) [reply companies/bigpoint.json]: [here](https://www.bigpoint.net/imprint/). That was before I updated the `sources`. It is generic fax, not a privacy related, so I guess the fax can be removed.

File: companies/bigpoint.json (hunk 4 above)
Comment (Richardds) [reply companies/bigpoint.json]: removed

File: companies/bigpoint.json (hunk 2 above)
Comment (Richardds) [reply companies/bigpoint.json]: fixed

File: companies/bigpoint.json (hunk 1 above)
Comment (Richardds) [reply companies/bigpoint.json]: fixed

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**